### DISCUSSION TITLE
Issues with dependency when installing packages from requirements.txt

### DISCUSSION THREAD

**Discussion by asimthapa** [TreyWW/MyFinances#154] [Q&A] [answered]
Hi,
I am trying to set up the project locally in venv. I keep getting issues with installing mysqlclient

- First I got mysql.h file not found, which I have to install manually and add it to path.
- Now, I am getting **fatal error LNK1181: cannot open input file 'mariadbclient.lib'**. That file is not in any of the path and I tried to look for that file onthe internet, but I couldn't find anything.

![image](https://github.com/TreyWW/MyFinances/assets/46013321/e7e74d5c-9e9f-4872-9e65-1d6c2a9191c9)

**Comment by TreyWW** [comment, marked as answer]
Hi @asimthapa,

If you plan on using MySQL as your database this package is a requirement to connect to it. To fix your error you'll need to install the MySQL wheel for your OS, I can provide the link to this if you need it. 

If you don't plan on using MySQL and prefer to use SQLite or Postgres then you can install packages from `gh_actions_requirements.txt`. That contains (well should contain) all dependencies apart from MySQL. 

Thanks for reaching out let me know if this doesn't work

**Reply by asimthapa** [reply]
thank you, I got it resolved. I was using python version 3.12 but looks like mysql wheel is only supported till v3.9 (yet). I changed my python version to 3.9 and it's working now.
